management/telemetry/telemetry-dashboards/telemetry-dashboards.md

```diff
@@ -0,0 +1,197 @@
+
+These four visuals show PPU consumption from different angles. The first focuses on *PPU per Configuration*, differentiating between SQL, KBC, and CDC credits. *PPU Daily* shows a breakdown of PPU 
+between component types over the period selected. *Component's Usage* is a treemap and provides an overview of how many jobs were executed for the components. You can drill down for a particular 
+configuration or project to see how many credits it consumed. The last visual also shows the credits consumed per component split between SQL, KBC, and CDC credits. 
+
+#### Top 10 
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-top10.png)
+
+Top 10 has two tables. The first is the *Top 10 Transformations* based on their consumption. It provides details about the runtime and input and output mapping (e.g., whether any filters were applied or 
+whether the output was incremental or full load). *Top 10 Component Configurations* shows the most expensive configurations and provides information on how long they ran for. 
+
+#### Significant changes (outliers)
+{: .image-popup}
+![Screenshot - Top 10](/management/telemetry/telemetry-dashboards/ac-project-overview-outliers.png)
+
+*Significant changes* shows outliers for projects, configurations, and users. The tables show the average PPU consumption over the previous 30 days for projects, configurations, or users that consumed
+50% more or 50% less on given days. It lists days on which a project billed more than 1 credit. The last column is the percentage of billed credits versus the average. 
+
+### Project Users
+The first section provides an *Overview* of users within a project. They can be filtered by branch type and specific user. It shows the user roles, activities, and how many joined the project 
+in the last seven days. 
+
+#### Overview
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-users-overview.png)
+
+The second section, *Activities*, summarizes the users in a project. 
+
+#### Activities
+{: .image-popup}
+![Screenshot - Activities](/management/telemetry/telemetry-dashboards/ac-project-users-activities.png)
+
+### Project Health
+This tab shows the overall health of a project. It can be filtered by branch (development vs. main). The *Overview* section focuses on flows and jobs. The next section focuses on trends for *Components*, 
+and the last section shows four individual metrics: Configurations without a Description*, *Flows without Error Notifications*, *Unused Configurations*, and *Unused Tables*.
+
+#### Overview
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-overview.png)
+
+#### Components
+{: .image-popup}
+![Screenshot - Project Users Overview](/management/telemetry/telemetry-dashboards/ac-project-health-components.png)
+
```

Comment:
This section shows more detailed view on the errors. *Error Jobs PPU* shows how many credits were consumed by jobs that ended in error, after click the drill down shows more details. *PPU Consumed by Error Jobs* shows the evolution over time. *Components Errors Trend* looks into the errors split by the component type over time. Last two graphs (*Components Errors* and *Components Error Ratio*) shows the same information on components error - first one in absolute values, second one in percentage.